app/config/config_test.yml

```diff
@@ -26,10 +26,6 @@ swiftmailer:
 
 # Doctrine Configuration
 doctrine:
-    dbal:
-        connections:
-            default:
-                dbname: "test_%database_name%"
```

Comment:
It shouldn't be removed, at the end, we will only trust doctrine configuration and remove everything related to  database outside this file :sweat_smile:

Reply:
Why? `%database_name%` variable should be modified for test environment and used (set in `doctrine.yml` for all environments).

Reply:
@PierreRambaud set as dynamically in `set_parameters.php` (https://github.com/PrestaShop/PrestaShop/pull/22823/files#diff-7ccf4e3f7361df4e8411e2fbf71e77b0d7620489d8f3cca67959993106a18b95R53), loaded in https://github.com/PrestaShop/PrestaShop/blob/1.7.7.3/app/config/config.yml#L2

Reply:
@PierreRambaud please rereview, this PR is important for me as it fixes #22840

Reply:
@PierreRambaud please confirm this impl. is correct